### PR TITLE
feat: governance UI hash preview

### LINKED ISSUE #984: Governance currently polls twice
**Describe the bug**
When you are checking out proposals the rpcs are getting bombed with requests.
The reason for that is:
a) fetching data extremely frequently (every 15s for sth that changes <100times in 5 days)
b) fetching everything twice (as there's two pollings)

This results in rate limits and slower response overall.
Therefore I suggest to:
- reduce rate (to 30s for now, but 1-2min would probably be still sufficient)
- reduce the double polling

### PR DESCRIPTION
**Pull request by sakulstra**
adds a hidden page which can be used to preview an ipfs hash. This is to prevent issues like the one with oneInch where a wrong ipfs hash was submitted.

- closes #984 #985

### CODE REVIEW

File: src/modules/governance/ProposalsList.tsx [hunk 1]
```diff
@@ -88,7 +88,7 @@ export function ProposalsList({ proposals: initialProposals }: GovernancePagePro
   }
 
   usePolling(fetchNewProposals, 60000, false, [proposals.length]);
-  usePolling(updatePendingProposals, 15000, false, [proposals.length]);
+  usePolling(updatePendingProposals, 30000, false, [proposals.length]);
```
Comment (sakulstra) [inline src/modules/governance/ProposalsList.tsx:91]: there's around 100 votes in 5 days... seems a bit unreasonable to refetch every 15s, even 30 seems unreasonable, but i guess small steps are better suited to test if someone complains.

File: pages/governance/proposal/index.governance.tsx [hunk 2]
```diff
@@ -19,34 +17,27 @@ export default function DynamicProposal() {
   const [proposal, setProposal] = useState<CustomProposalType>();
   const [ipfs, setIpfs] = useState<IpfsType>();
 
-  async function updateProposal() {
-    const { values, ...rest } = await governanceContract.getProposal({ proposalId: id });
-    setProposal(await enhanceProposalWithTimes(rest));
+  async function initialize() {
```
Comment (sakulstra) [inline pages/governance/proposal/index.governance.tsx:20]: this was a bit unnecessary complex as the proposal and ipfs fetching where made independent although they belong together.

File: pages/governance/proposal/index.governance.tsx [hunk 3]
```diff
@@ -19,34 +17,27 @@ export default function DynamicProposal() {
...
-    const newIpfs = {
-      id,
-      originalHash: proposal.ipfsHash,
-      ...(await getProposalMetadata(proposal.ipfsHash, governanceConfig?.ipfsGateway)),
-    };
-    setIpfs(newIpfs);
-  }
-
-  // poll every 10s
-  usePolling(
```
Comment (sakulstra) [inline pages/governance/proposal/index.governance.tsx:38]: this polling effectively resulted in double fetching of everything as ProposalPage is polling as well

**Comment by defispartan** [comment]
Pls create an issue as well @sakulstra

**Comment by sakulstra** [comment]
@defispartan created #984 and #985 for this.

**Comment by sakulstra** [comment]
@defispartan sorry missed your comment, looking forward to see this merged.

**Review by defispartan — APPROVED** [review]

**Review by grothem — APPROVED** [review]

**Pull request merged**